include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -859,6 +859,50 @@ class MGTransferGlobalCoarsening : public dealii::MGTransferBase<VectorType>
   build(const std::function<void(const unsigned int, VectorType &)>
           &initialize_dof_vector);
 
+  /**
+   * Default constructor.
+   *
+   * @note See also MGTransferMatrixFree.
+   */
+  MGTransferGlobalCoarsening() = default;
+
+  /**
+   * Constructor with constraints. Equivalent to the default constructor
+   * followed by initialize_constraints().
+   *
+   * @note See also MGTransferMatrixFree.
+   */
+  MGTransferGlobalCoarsening(const MGConstrainedDoFs &mg_constrained_dofs);
```

Comment:
should we move all the constructors to the top of the class?

Reply:
All the functions related to the functionalities of `MGTransferMatrixFree` are grouped together.